features/users/logging_out.feature

```diff
@@ -5,9 +5,25 @@ Feature: User Logging out
   Scenario: Logging out successfully
     Given a configuration of:
     """
-      ActiveAdmin.register Post
+      ActiveAdmin.setup do |config|
+        config.logout_link_method = :get
+      end
     """
-    And I am logged in
+     And I am logged in
```

Comment:
At least by looking at the existing logging in and logging out features, we seem to be using left alignment for cucumber keywords (`When`, `And`, `Given`, `Then`). Yesterday I starred a [gherkin-lint](https://github.com/vsiakka/gherkin-lint) project that might (or might not) help enforcing this, but for now let's keep the existing style so we can keep the diff free of style changes.

Reply:
I totally understand, I also prefer to keep things consistent 👍